syntax = "proto3";

package reading_example;

message getMeDataRequest{
	string col_name = 1;
}

message getMeDataResponse{
	string df = 1;
}

service Computator{
	rpc getMeInformation (getMeDataRequest) returns (getMeDataResponse) {}
}